src/Microsoft.Azure.WebJobs.Host/Executors/IFunctionInstance.cs

```diff
@@ -22,6 +22,8 @@ public interface IFunctionInstance
         IFunctionInvoker Invoker { get; }
 
         FunctionDescriptor FunctionDescriptor { get; }
+
+        int RetryCount { get;  }
```

Comment:
This would be a breaking change to a public interface

Reply:
Fixed.